target/linux/pistachio/creator-platform-all-cascoda.config

```diff
@@ -0,0 +1,20 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_PACKAGE_uboot-pistachio_marduk=y
+CONFIG_ALL=y
+CONFIG_IB=y
+CONFIG_SDK=y
+CONFIG_MAKE_TOOLCHAIN=y
+CONFIG_KERNEL_KALLSYMS=y
+CONFIG_IMAGEOPT=y
+CONFIG_LOCALMIRROR="http://downloads.creatordev.io/pistachio/marduk/dl"
+CONFIG_VERSIONOPT=y
+CONFIG_VERSION_DIST="OpenWrt"
+CONFIG_VERSION_NICK="Ci40All"
+CONFIG_VERSION_NUMBER="dev"
```

Comment:
this should be 0.9.5, as there is no relation of "dev" in context of CreatorDev/openwrt repo